golem-debugging-service/src/from_value.rs

```diff
@@ -657,13 +706,34 @@ impl FromValue for GolemError {
 
                         let details = String::from_value(&values[0])?;
 
-                        Ok(GolemError::Unknown { details })
+                        Ok(WorkerExecutorError::Unknown { details })
                     }
 
                     _ => Err("Failed to get GolemError. Not a Record".to_string()),
                 },
-                (22, None) => Ok(GolemError::ShardingNotReady),
-                (23, Some(error)) => match error.deref() {
+                (22, None) => Ok(WorkerExecutorError::ShardingNotReady),
```

Comment:
tbh, I'm not quite sure how these are being used.

Will dig into it some other time as it seems quite brittle